### PR TITLE
added scraper for lewrubens.com

### PR DESCRIPTION
**Pull request by deezero**
added xpath scraper for lewrubens.com

### CODE REVIEW

File: scrapers/LewRubens.yml [hunk 1]
```diff
@@ -0,0 +1,14 @@
...
+    url:
+      - lewrubens.com/x-new/new-set.php?setid=
+    scraper: sceneScraper
+
+xPathScrapers:
+  sceneScraper:
+    scene:
+      Title: //meta[@name="twitter:title"]/@content
+      Details: //meta[@name="twitter:description"]/@content
+      Image: //meta[@name="twitter:image"]@content
```
Comment (bnkai) [inline scrapers/LewRubens.yml:14]: typo for the image part and the last updated comment is missing
```suggestion
      Image: //meta[@name="twitter:image"]/@content
# Last Updated February 6, 2021
```

**Comment by nocrad349** [comment]
Maybe add the studio name? Something like:

```
Studio:
  Name:
    fixed: Lew Rubens
```

Or is it Lew Rubens Productions?

**Comment by deezero** [comment]
@bnkai Fixed the xPath, but I've noticed an issue with unescaped quotes.

> Maybe add the studio name? Something like:
> 
> ```
> Studio:
>   Name:
>     fixed: Lew Rubens
> ```
> 
> Or is it Lew Rubens Productions?

The logo on the website says "Lew Rubens (.com)", but I'm sure I've seen videos that say "Lew Rubens Productions".

File: scrapers/LewRubens.yml (hunk 1 above)
Comment (deezero) [reply scrapers/LewRubens.yml]: Fixed in a new commit.

**Comment by deezero** [comment]
Not sure how to get around:

`<meta name="twitter:description" content="some text, then unescaped quotes "person said this" rest of description here">`

Firefox notes it's a malformed line.

**Comment by deezero** [comment]
I'm selecting the body description now, seems to fix it, but adds a "XX:XX video" prefix to the description.

**Comment by deezero** [comment]
Ok, I modified the selector to ignore the "Share" button and "video" text, although there is an edge case where if the description ends in "video" it will be excluded as well.

**Comment by bnkai** [comment]
It seems that the details in most samples i looked where inside spans
```yaml
      Details: 
        selector: //div[@class="album-details"]/div/p//span/text()
        concat: " "
```
Something like the above seemed to work (didnt test many scenes though)

**Comment by nocrad349** [comment]
Only the newest videos (up to page 5ish of the updates) have the span tags. Older videos don't have the span tags.

The big problem is that they seem to copy/paste from Word (I've seen a few class="MsoNormal"). So there are a lot of different cases. I've scraped all Lew Rubens scenes in my collection (46), resulting in the following types of cases:

multiple \<p\> tags with plain description text:
https://lewrubens.com/x-new/new-set.php?setid=44911

description text with br-tags inside  \<p\>:
https://lewrubens.com/x-new/new-set.php?setid=43506

multiple \<span\> inside  \<p\>:
https://lewrubens.com/x-new/new-set.php?setid=105755

multiple \<strong\> and plain description text inside \<p\>:
https://lewrubens.com/x-new/new-set.php?setid=84302

"all" of the above:
https://lewrubens.com/x-new/new-set.php?setid=93814
https://lewrubens.com/x-new/new-set.php?setid=93811

The following works for these cases, with a few small problems.

```yaml
    Details:
      selector: //div[@class="album-details"]/div/p[@style="border:0px solid red;"]/following-sibling::p//text()
      concat: " "
```

The problems are:
* \<br\> replaced with a space intead of a newline
* Words that are split by tags get extra spaces. For instance "\<strong\>Sasha p\</strong\>\<strong\>eeks\</strong\>" becomes "Sasha p eeks".

**Comment by bnkai** [comment]
They dont seem to have a consistent formatting. If no one has any better alternative i think the one proposed by @nocrad349 is slightly better than the one in PR.
@deezero can you edit and replace or add the proposed one as a comment?

**Pull request merged**